spec/lib/services/swagger-api-service-spec.js

```diff
@@ -502,6 +502,17 @@ describe('Services.Http.Swagger', function() {
             });
         });
 
+        it('should throw 500 on invalid template', function() {
+            res.body = { message: "foo" };
+            return renderer(req, res, 'foo', mockNext)
+            .then(function() {
+                expect(mockNext).to.be.calledWithMatch({status: 500});
+            },
+            function(err) {
```

Comment:
I think this error callback is not necessary. If remove this error callback, the unit-test will still be marked as failed since the promise is rejected.